fairlearn/preprocessing/_information_filter.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import check_array
+from sklearn.utils.validation import check_is_fitted
+from sklearn.preprocessing import StandardScaler
+
+
+def _scalar_projection(vec, unto):
```

Comment:
Do you mean `onto` rather than `unto` ? In any case, a longer name might be helpful.